Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -19,99 +19,99 @@ This project is released under the MIT License.
-  behaviorContent.SetAttribute("acceleration", 1500);
-  behaviorContent.SetAttribute("deceleration", 1500);
-  behaviorContent.SetAttribute("maxSpeed", 250);
-  behaviorContent.SetAttribute("jumpSpeed", 600);
-  behaviorContent.SetAttribute("jumpSustainTime", 0.2);
-  behaviorContent.SetAttribute("ignoreDefaultControls", false);
-  behaviorContent.SetAttribute("slopeMaxAngle", 60);
-  behaviorContent.SetAttribute("canGrabPlatforms", false);
-  behaviorContent.SetAttribute("canGrabWithoutMoving", true);
-  behaviorContent.SetAttribute("yGrabOffset", 0);
-  behaviorContent.SetAttribute("xGrabTolerance", 10);
-  behaviorContent.SetAttribute("useLegacyTrajectory", false);
-  behaviorContent.SetAttribute("canGoDownFromJumpthru", true);
+	gd::SerializerElement& behaviorContent) {
+	
+	behaviorContent.SetAttribute("ignoreDefaultControls", false);
+	
+	behaviorContent.SetAttribute("gravity", 1000);
+	behaviorContent.SetAttribute("jumpSpeed", 600);
+	behaviorContent.SetAttribute("jumpSustainTime", 0.2);
+	behaviorContent.SetAttribute("maxFallingSpeed", 700);
+	behaviorContent.SetAttribute("useLegacyTrajectory", false);
+	
+	behaviorContent.SetAttribute("ladderClimbingSpeed", 150);
+	
+	behaviorContent.SetAttribute("canGrabPlatforms", true);
+	behaviorContent.SetAttribute("canGrabWithoutMoving", true);
+	behaviorContent.SetAttribute("xGrabTolerance", 10);
+	behaviorContent.SetAttribute("yGrabOffset", 0);
+	
+	behaviorContent.SetAttribute("canGoDownFromJumpthru", true);
+	behaviorContent.SetAttribute("acceleration", 1500);
+	behaviorContent.SetAttribute("deceleration", 1500);
+	behaviorContent.SetAttribute("maxSpeed", 250);
+	behaviorContent.SetAttribute("slopeMaxAngle", 60);
 }
 
-#if defined(GD_IDE_ONLY)
-std::map<gd::String, gd::PropertyDescriptor>
-PlatformerObjectBehavior::GetProperties(
-    const gd::SerializerElement& behaviorContent) const {
-  std::map<gd::String, gd::PropertyDescriptor> properties;
-
-  properties[_("Gravity")].SetGroup(_("Jump")).SetValue(
-      gd::String::From(behaviorContent.GetDoubleAttribute("gravity")));
-  properties[_("Jump speed")].SetGroup(_("Jump")).SetValue(
-      gd::String::From(behaviorContent.GetDoubleAttribute("jumpSpeed")));
-  properties["jumpSustainTime"]
+#
+if defined(GD_IDE_ONLY)
+std::map < gd::String, gd::PropertyDescriptor >
+    PlatformerObjectBehavior::GetProperties(
+        const gd::SerializerElement& behaviorContent) const {
+        std::map < gd::String, gd::PropertyDescriptor > properties;
+
+    properties[_("Default controls")]
+      .SetValue(behaviorContent.GetBoolAttribute("ignoreDefaultControls") ? "false" : "true")
+      .SetType("Boolean");
+
+    properties[_("Gravity")]
+      .SetGroup(_("Jump"))
+      .SetValue(gd::String::From(behaviorContent.GetDoubleAttribute("gravity")));
+    properties[_("Jump speed")]
+      .SetGroup(_("Jump"))
+      .SetValue(gd::String::From(behaviorContent.GetDoubleAttribute("jumpSpeed")));
+    properties[_("jumpSustainTime")]
```

Comment:
Unfortunately that should be the reverse:

Property internal names should not be translated. `SetLabel(_("Translatable name"))` should be used instead.

Reply:
(otherwise, you ask translators on Crowdin to translate "jumpSustainTime", and this will be desynchronized with `UpdateProperty` method that was not updated)